extensions/amp-form/0.1/amp-form.js

```diff
@@ -600,7 +600,9 @@ export class AmpForm {
           presubmitPromises,
           SUBMIT_TIMEOUT
         ).then(
-          () => this.handlePresubmitSuccess_(trust),
+          () => {
```

Comment:
Nit: Is it the case that this might not be necessary since arrow functions have [implicit return](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Function_body)?